package.json

```diff
@@ -60,7 +61,7 @@
     "test:unit": "jest --config=./jest.coverage.config.js test/unit",
     "test:watch": "jest --coverageReporters none --watch test/unit",
     "validate": "componentsjs-compile-config urn:solid-server:default:Initializer -c config/default.json -f > /dev/null",
-    "version": "manual-git-changelog onversion",
+    "upgrade-config": "ts-node ./scripts/upgradeConfig.ts",
```

Comment:
We should also do this for `createAccountCredentials` instead of the build that happens first now in the schedule workflow. So feel free to also update that if you're looking for things to do.